### PR TITLE
Fix schema references in parameters

### PR DESCRIPTION
**Pull request by jangnezda**
Fixes [issue #52](https://github.com/api-hogs/bureaucrat/issues/52), so that the generated markdown file properly includes references to schema models in operation parameters.

### CODE REVIEW

File: lib/bureaucrat/swagger_slate_markdown_writer.ex [hunk 1]
```diff
@@ -228,15 +228,15 @@ defmodule Bureaucrat.SwaggerSlateMarkdownWriter do
   tag roughly corresponds to a phoenix controller, eg "Users"
   records_by_operation_id are the examples collected during tests, grouped by operationId (Controller.action)
   """
-  def write_operations_for_tag(file, tag, records_by_operation_id, swagger) do
+  def write_operations_for_tag(file, swagger, tag, records_by_operation_id, swagger) do
```
Comment (OpakAlex) [inline lib/bureaucrat/swagger_slate_markdown_writer.ex]: why double `swagger ` here?

File: lib/bureaucrat/swagger_slate_markdown_writer.ex [hunk 2]
```diff
@@ -228,15 +228,15 @@ defmodule Bureaucrat.SwaggerSlateMarkdownWriter do
...
     tag_details = swagger["tags"] |> Enum.find(&(&1["name"] == tag))
 
     file
     |> puts("# #{tag}\n")
     |> puts("#{tag_details["description"]}\n")
 
     Enum.each(records_by_operation_id, fn {operation_id, records} ->
-      write_action(file, operation_id, records, swagger)
+      write_action(file, swagger, operation_id, records, swagger)
```
Comment (OpakAlex) [inline lib/bureaucrat/swagger_slate_markdown_writer.ex]: same

File: lib/bureaucrat/swagger_slate_markdown_writer.ex (hunk 1 above)
Comment (jangnezda) [reply lib/bureaucrat/swagger_slate_markdown_writer.ex]: Oops, totally overlooked it. Fixed now.

**Comment by OpakAlex** [comment]
Thanks!

**Pull request merged**